src/common/svgs/jira.svg

```diff
@@ -0,0 +1 @@
+<svg height="2500" preserveAspectRatio="xMidYMid" width="2500" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 -30.632388516510233 255.324 285.95638851651023"><linearGradient id="a"><stop offset=".18" stop-color="#0052cc"/><stop offset="1" stop-color="#2684ff"/></linearGradient><linearGradient id="b" x1="98.031%" x2="58.888%" xlink:href="#a" y1=".161%" y2="40.766%"/><linearGradient id="c" x1="100.665%" x2="55.402%" xlink:href="#a" y1=".455%" y2="44.727%"/><path d="M244.658 0H121.707a55.502 55.502 0 0 0 55.502 55.502h22.649V77.37c.02 30.625 24.841 55.447 55.466 55.467V10.666C255.324 4.777 250.55 0 244.658 0z" fill="#2684ff"/><path d="M183.822 61.262H60.872c.019 30.625 24.84 55.447 55.466 55.467h22.649v21.938c.039 30.625 24.877 55.43 55.502 55.43V71.93c0-5.891-4.776-10.667-10.667-10.667z" fill="url(#b)"/><path d="M122.951 122.489H0c0 30.653 24.85 55.502 55.502 55.502h22.72v21.867c.02 30.597 24.798 55.408 55.396 55.466V133.156c0-5.891-4.776-10.667-10.667-10.667z" fill="url(#c)"/></svg>
```

Comment:
**Question:** Out of curiosity, where did you get this SVG? If possible, we should use the official one from their press kit -- there should be a blue version of the icon in the zip here: https://assets.ctfassets.net/8j5aqoy0ts8s/1y9Zan62VaterUggPDIDAx/7537cd8851766dfce84d0785dc8200d7/Jira.zip

I just ask because I've been trying to be cautious with official org logos and to use the official press kit when possible 😄 , and this one seemed a little different from that one in the press kit! (And please ignore if this is from there and I'm making the comparison incorrectly! 🙇 )

Reply:
> I just ask because I've been trying to be cautious with official org logos and to use the official press kit when possible

That's a valid point! I got it from the first searches on Google. I've replaced the SVG to use the official one 🙇🏻‍♀️ Fix can be found here: [3beaeed](https://github.com/Automattic/bugomattic/pull/111/commits/3beaeed3e0978ae9eda3a66cd8c38acef13d4f47)